### DISCUSSION TITLE
Add ReviewNB for to compare notebooks (ipynb)

### DISCUSSION THREAD

**Discussion by wildintellect** [cloudnativegeo/cloud-optimized-geospatial-formats-guide#57] [Ideas]
I attempted to add https://github.com/apps/review-notebook-app/installations/new so we can diff notebooks on PRs but it doesn't seem to show up yet in PRs. Not sure if there's an organization setting @jedsundwall that needs to allow the application.

**Comment by wildintellect** [comment]
It's working now on any PRs with ipynb, just needed some patience.

**Comment by jedsundwall** [comment]
Yeah sorry I should have told you I approved your request. Not sure if you got a notification about that.
